.gitignore

```diff
@@ -1 +1,2 @@
 codecept.phar
+node_modules
```

Comment:
This also helps if someone runs npm install in the dev-lib folder unintentionally, as we're not supposed to add any node_modules to the repo, even if by mistake.